lang/en/tool_pluginskel.php

```diff
@@ -25,153 +25,359 @@
+$string['emptypluginname'] = 'Plugin name not specified';
+$string['emptyrecipecontent'] = 'Empty recipe';
+
+$string['enrol_features_allow_enrol'] = 'Allow enrol';
+$string['enrol_features_allow_enrol_help'] = 'Allow user enrolment from other plugins by calling the function enrol_user(). A corresponding "enrol" capability must also be defined.';
+$string['enrol_features_allow_enrol_link'] = 'https://docs.moodle.org/dev/Enrolment_plugins';
+$string['enrol_features_allow_unenrol'] = 'Allow unenrol';
+$string['enrol_features_allow_unenrol_help'] = 'Allow other plugins to unenrol everybody. A corresponding "unenrol" capability must also be defined.';
+$string['enrol_features_allow_unenrol_link'] = $string['enrol_features_allow_enrol_link'];
+$string['enrol_features_allow_unenrol_user'] = 'Allow unenrol user';
+$string['enrol_features_allow_unenrol_user_help'] = 'Allow other plugins to unenrol a specific user. A corresponding "unenrol" capability must also be defined.';
+$string['enrol_features_allow_unenrol_user_link'] = $string['enrol_features_allow_enrol_link'];
+$string['enrol_features_allow_manage'] = 'Allow manage';
+$string['enrol_features_allow_manage_help'] = 'Allow other plugins to manually modify the user enrolments.';
+$string['enrol_features_allow_manage_link'] = $string['enrol_features_allow_enrol_link'];
+
+$string['events'] = 'Events';
+$string['events_eventname'] = 'Event name';
+$string['events_eventname_help'] = 'The name of the created event.';
+$string['events_eventname_link'] = 'https://docs.moodle.org/dev/Event_2';
+$string['events_extends'] = 'Extends';
+$string['events_extends_help'] = 'The name of the base event that the event extends.';
+$string['events_extends_link'] = $string['events_eventname_link'];
+
+$string['features_install'] = 'Install';
+$string['features_install_help'] = 'Generate the file db/install.php';
+$string['features_license'] = 'License';
+$string['features_license_help'] = 'Generate the LICENSE.md file with the text of the GPL3 license.';
+$string['features_readme'] = 'Readme';
+$string['features_readme_help'] = 'Generate the README.md file.';
+$string['features_settings'] = 'Settings';
+$string['features_settings_help'] = 'Generate the settings.php file.';
+$string['features_uninstall'] = 'Uninstall';
+$string['features_uninstall_help'] = 'Generate the file db/uninstall.php.';
+$string['features_upgrade'] = 'Upgrade';
+$string['features_upgrade_help'] = 'Generate the file db/upgrade.php.';
+
+$string['generateskel'] = 'Generate plugin skeleton';
+$string['generalhdr'] = 'General';
+
+$string['lang_strings'] = 'Lang strings';
+$string['lang_strings_id'] = 'Id';
+$string['lang_strings_id_help'] = 'The id of the language string.';
+$string['lang_strings_id_link'] = 'https://docs.moodle.org/dev/String_API';
+$string['lang_strings_text'] = 'Text';
+$string['lang_strings_text_help'] = 'The value of the language string.';
+$string['lang_strings_text_link'] = $string['lang_strings_id_link'];
```

Comment:
Please do not do these things. String files should be considered as data files. There are tools that have to parse plugins' string files without evaluating them - https://docs.moodle.org/dev/Plugin_contribution_checklist#Strings

It is recommended to simply copy the string value in this case.